### ISSUE TITLE
[0.6.2-r7] Login fail: No airpair.com user found linked with GitHub login sphinx-solution

### ISSUE THREAD

**Issue by sphinx-solution** [airpair/author.airpair.com#3]

**Comment by jkresner** [comment]
*\* 

Pretty clear:

![screen shot 2016-02-21 at 2 08 49 am](https://cloud.githubusercontent.com/assets/979542/13197322/6b630b54-d83f-11e5-8305-f118ffac1a08.png)

**Comment by etagwerker** [comment]
@jkresner Right, it is pretty clear, but users will still get to this page: 

![airpair-error](https://cloud.githubusercontent.com/assets/17584/13813199/02949556-eb5f-11e5-9bca-f2172307f1cd.png)

The user experience would be more friendly if the error page showed something like the screenshot that you posted here: https://github.com/airpair/author.airpair.com/issues/3#issuecomment-186621910

I missed the "*\* If you don't have an airpair.com account ..." message too.

**Comment by jkresner** [comment]
@etagwerker will take a look, thought that experience was resolved might be older version of some code still deployed. thanks.

**Comment by bora89** [comment]
guys, still does not work, I can not login with github

**Comment by momoadeli** [comment]
@jkresner Getting same error.  Can't login/register on airpair.
